module/userProjects/helper.js

```diff
@@ -2426,41 +2431,49 @@ module.exports = class UserProjectsHelper {
                     }
                 });
 
-                if( appName !== "" ) {
+                createProject["appInformation"] = {};
+                if (appName !== "") {
                     createProject["appInformation"]["appName"] = appName;
                 }
 
-                if( appVersion !== "" ) {
+                if (appVersion !== "") {
                     createProject["appInformation"]["appVersion"] = appVersion;
                 }
 
+                createProject["lastDownloadedAt"] = new Date();
+
+                if (data.profileInformation) {
+                    createProject.userRoleInformtion = data.profileInformation;
```

Comment:
@aman-tunerlabs , @deepakr123 are we storing mongo roleId in userRoleInformtion ?

Reply:
@deepakr123  Can you confirm on this please ?